test/functions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class functions extends TestCase
+{
+    public function testDisplayAnimalsSuccess()
```

Comment:
better name would be `testDisplayAnimalsSuccessString` and the other one `testDisplayAnimalsSuccessCorrectOutput`